### DISCUSSION TITLE
Cannot find csrf issue

### DISCUSSION THREAD

**Discussion by Hruda-Rockey10** [woshiluo/cf-tool#4] [Q&A]
@woshiluo can u plz help me with the csrf issue ..
after clicking on submit it shows ( cannot find csrf)

**Comment by woshiluo** [comment]
Honestly, I haven't got a idea which is good enough to bypass codeforces's checking. I will try on it if a have time, but it will need time.

But If you need, you can use https://mirror.codeforces.com/, which is official codeforces mirror, but this site's checking haven't been as strong as original, it's a good temporary solution.

**Comment by Hruda-Rockey10** [comment]
i tried with mirror website it says cf tool cant parse using that mirror website

**Reply by woshiluo** [reply]
Can you give me some screenshots? I currently use mirror website and it works well.

**Comment by Hruda-Rockey10** [comment]
<img width="482" alt="wq" src="https://github.com/user-attachments/assets/976710ba-b0e4-4f7f-a5ae-99b289dce935">

@woshiluo

**Reply by woshiluo** [reply]
It seems that you are not using cf-tool directly, I may need more information like which command do you run.

**Reply by Hruda-Rockey10** [reply]
what information d u need?

**Reply by Hruda-Rockey10** [reply]
i used competitive companion to parse test cases
and while submitting i was getting this error

**Reply by woshiluo** [reply]
I'm kind of busy recently， i'll check it if i have time.
